json2html/jsonconv.py

```diff
@@ -28,25 +28,34 @@
 
 if sys.version_info[:2] < (3, 0):
     text = unicode
-    text_types = [unicode, str]
+    text_types = (unicode, str)
 else:
     text = str
-    text_types = [str]
+    text_types = (str,)
 
 class Json2Html:
-    def convert(self, json="", table_attributes='border="1"', clubbing=True, encode=False):
+    def convert(self, json="", table_attributes='border="1"', clubbing=True, encode=False, escape=True):
```

Comment:
This is totally fine but I was thinking of having a config instead of passing so many arguments.
Like: def convert(self, json="", config)

It would be easy-to-use and improve readability. I'll address this in a separate PR, no worries :)